src/test/java/com/google/javascript/gents/singleTests/comments.js

```diff
@@ -44,3 +44,28 @@ var x;
 
 /** @export {number} */
 let m = 4;
+
+// head comment
+if (m) {
+  // comment at the bottom of if block
+}
+else {
+  // comment at the bottom of else block
+}
+
+var a = function() {
+  return {
+    b: function() {}
+  }
+};
+var c = function() {};
+
+// comment before GETPROP
+a(
+// comment in GETPROP
```

Comment:
I am still confused why this is in GETPROP and not in FUNC_CALL (or something of sorts).